File: 1.INICIANTE/EX_1010.c
#include<stdio.h>

int p1=0, qtd_p1=0, p2=0, qtd_p2=0;
float valor_p1 = 0, valor_p2=0, total=0;

main()
{
	scanf("%d %d %f",&p1, &qtd_p1, &valor_p1);
	scanf("%d %d %f",&p2, &qtd_p2, &valor_p2);
	total = qtd_p1*valor_p1 + qtd_p2*valor_p2;
	printf("VALOR A PAGAR: R$ %.2f\n",total);
	return 0;
	
}

File: 1.INICIANTE/EX_1010.py
codigo_p1, qtd_p1, valor_p1 =input().split()

codigo_p2, qtd_p2, valor_p2 = input().split()

total = int(qtd_p1)*float(valor_p1) + int(qtd_p2)*float(valor_p2)

print(f'VALOR A PAGAR: R$ {total:.2f}')